applications/nrf5340_audio/README.rst

```diff
@@ -898,7 +898,9 @@ For example, to build the application using the script for the application core
 
    python buildprog.py -c app -b debug -d both
 
-The build files are saved in the :file:`build` directory.
+This command can be added as an alias, such that the script can be called from other directories as well.
```

Comment:
What do you mean exactly by "This command can be added as an alias"?
I can now e.g. call buildprog.py from the nrf5340_audio folder, which is nice.